libparsec/crates/client/tests/unit/certif/encrypt_for_realm.rs

```diff
@@ -0,0 +1,119 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+use libparsec_client_connection::test_register_send_hook;
+use libparsec_protocol::authenticated_cmds;
+use libparsec_tests_fixtures::prelude::*;
+use libparsec_types::prelude::*;
+
+use crate::certif::CertifEncryptForRealmError;
+
+use super::utils::certificates_ops_factory;
+
+#[parsec_test(testbed = "minimal")]
+async fn ok(env: &TestbedEnv) {
+    let (env, realm_id) = env.customize_with_map(|builder| {
+        builder
+            .new_realm("alice")
+            .then_do_initial_key_rotation()
+            .map(|event| event.realm)
+    });
+    let alice = env.local_device("alice@dev1");
+    let ops = certificates_ops_factory(&env, &alice).await;
+
+    ops.add_certificates_batch(
```

Comment:
use `builder.certificates_storage_fetch_certificates("alice@dev1");` instead

Reply:
You can suggest code directly in the comment, see [Adding comments to a PR](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#adding-comments-to-a-pull-request) at the step `6`